helper/helper.go

```diff
@@ -418,3 +449,23 @@ func decryptAttachment(
 
 	return message, nil
 }
+
+func createPublicKeyRing(
+	publicKey string,
+) (publicKeyRing *crypto.KeyRing, err error) {
+	publicKeyObj, err := crypto.NewKeyFromArmored(publicKey)
+
+	if publicKeyObj.IsPrivate() {
+		publicKeyObj, err = publicKeyObj.ToPublic()
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if err != nil {
```

Comment:
Check the error first, `publicKeyObj` might be nil and panic

Reply:
Right, Thanks